### PR TITLE
Fixed typos and changed some words for better tone.

### PR DESCRIPTION
**Pull request by ArchaeotheriumSapienter**
Changed some typographical errors in some Python files and changed the word "some things" to more specific wordings for a more specific and better tone of communication.

### CODE REVIEW

File: sphinx/source/docs/first_steps/examples/first_steps_4_grid_lines.py [hunk 1]
```diff
@@ -15,10 +15,10 @@
 # add a renderer
 p.line(x, y, line_color="green", line_width=2)
 
-# change just some things about the x-grid
+# change line color of the x-grid
 p.xgrid.grid_line_color = "red"
 
-# change just some things about the y-grid
+# change grid line attributes of the y-grid
```
Comment (bryevdv) [inline sphinx/source/docs/first_steps/examples/first_steps_4_grid_lines.py:21]: cc @tcmetzger FWIW the wording here with "just" was intentional, I was trying to emphasize that the first line *only* affected the x-axis, while the second *only* affected the y-axis.

Comment (ArchaeotheriumSapienter) [reply sphinx/source/docs/first_steps/examples/first_steps_4_grid_lines.py:21]: I see, perhaps I could change it by using the word _only_, omitting the _just_

**Review by bryevdv — APPROVED** [review]

**Comment by bryevdv** [comment]
Thanks @ArchaeotheriumSapienter FYI I have changed the target branch to `branch-3.0` since that  is now the default.

**Comment by bryevdv** [comment]
@ArchaeotheriumSapienter can you rebase on `branch-3.0` and and force push an update? There is an extra commit creeping in after changing the target

**Comment by ArchaeotheriumSapienter** [comment]
Will do! I just encountered a problem after rebasing to `branch-3.0` where all of the past commits after `branch-2.4` are going to be included if I pushed so It'll probably take a while :)

**Comment by bryevdv** [comment]
@ArchaeotheriumSapienter submitting a brand new PR on a new branch would also be fine, if that is simpler.

**Comment by ArchaeotheriumSapienter** [comment]
Sure! I'll just create a new branch and close this pull request :)

**Pull request closed without merging**